.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,27 @@
+// For format details, see https://aka.ms/vscode-remote/devcontainer.json or this file's README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.106.0/containers/docker-existing-dockerfile
+{
+    "name": "Object Store",
+
+    "dockerComposeFile": "docker-compose.yml",
+    "service": "tests",
+    "workspaceFolder": "/realm-object-store",
+
+    // Set *default* container specific settings.json values on container create.
+    "settings": { 
+        "terminal.integrated.shell.linux": null,
+        "cmake.configureSettings": {
+            "REALM_ENABLE_AUTH_TESTS": "ON",
+            "REALM_MONGODB_ENDPOINT": "http://mongodb-realm:9090",
+            "REALM_STITCH_CONFIG": "/realm-object-store/tests/mongodb/stitch.json",
```

Comment:
does this absolute path actually work?

Reply:
Yes, that's how the workspace is mounted inside the container: https://github.com/realm/realm-object-store/pull/962/files#diff-682a74ffaa0a92339fe8ba23a3fc1febR7